syntax = "proto2";
message Enterprise 
{
    required int32 year_founded = 1; // Год основания
    required string legal_address = 2; // Юридический адрес
    required string name = 3; // Название

    optional string activity_type = 4; // Род деятельности
    optional bool foreign_economic_activity = 5; // Осуществление внешнеэкономической деятельности
}
